pytorch/training/docker/2.0/py3/cu121/Dockerfile.ec2.gpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "org.apache.ant:ant": [
```

Comment:
Is there a way to fix this by upgrading the mamba version or anything like that?

Reply:
I updated version to latest but it is not fixing it. That dir seems to be coming from some other package, need more deep dive to see after each package installation

Reply:
will recheck once dockerfile is ready

Reply:
i still see it, so keep the file